wire-ios/Wire-iOS/Sources/UserInterface/MainController/ZClientViewController.swift

```diff
@@ -417,37 +417,15 @@ final class ZClientViewController: UIViewController {
     }
 
     // MARK: - Animated conversation switch
-    func dismissAllModalControllers(callback: Completion?) {
-        let dismissAction = {
-            if let rightViewController = self.mainSplitViewController.viewController(for: .secondary),
```

Comment:
Any VC should be presented via `mainCoordinator.present` so that the main coordinator can dismiss it when it needs to perform navigation.